doc/source/tutorials/mutations_and_modifications.rst

```diff
@@ -0,0 +1,49 @@
+===========================
+Mutations and modifications
+===========================
+
+Martinize2 facilitates a powerful syntax for defining mutations and modifications to your input protein structure.
+Here we'll look at some of the options for how your protein can be changed during the coarse graining process. While
+chemically and structurally the two processes are different, the command line options share the same syntax.
```

Comment:
The rest of this tutorial only deals with mutations. And I think it would be worthwhile to add a note/warning/... that martinize2 does not generate coordinates, so that if you mutate a glycine to phenylalanine you're going to have a NaN day